### ISSUE TITLE
Install Cronicle on Windows

### ISSUE THREAD

**Issue by caneryalcin** [jhuckaby/Cronicle#446]
I am curious about cronicle but I couldn't way to install cronicle on windows.

If there is a way could you please informe me.

**Comment by mikeTWC1984** [comment]
No, it won't work on windows. It's using things like symlinks and linux style permissions/execution. As far as I remember it also uses ps utility to grab server stats. Probably something else. But it sounds like it totally doable to adopt this for windows though. I think the new version will use some cross platform module for stats, and symlinks is easy to replace with file copy. 

I used to run bunch of powershell scripts on windows server. I was evaluating a tool which is now is called  "Powershell Universal" for job scheduling, but cronicle is much better (to my mind) and free. I was able to port everything to powershell core (which can run on linux). So in case you are not dealing with some windows-only stuff you can probably do the same

**Comment by mcfetz** [comment]
My solution for this problem is running an ssh-server on the windows host. Then I can connect from cronicle with ssh to the host and execute the command remotly. Not the sleekest solution. But it works. ;-)

**Comment by caneryalcin** [comment]
I may have found a solution. Just use cronicle on docker.  You can use docker on windows and also docker commands on powershell. 
https://hub.docker.com/r/bluet/cronicle-docker via this link I installed  image of cronicle and initialized. In the link it shows how you will install image and run.

**Comment by KieranDevvs** [comment]
> I may have found a solution. Just use cronicle on docker. You can use docker on windows and also docker commands on powershell. https://hub.docker.com/r/bluet/cronicle-docker via this link I installed image of cronicle and initialized. In the link it shows how you will install image and run.

I need to execute some old .NET framework binaries that are windows only so running them under a POSIX like OS isn't an option for me. Would really like to see this get windows adoption!

**Comment by mikeTWC1984** [comment]
If anyone is still interested, I was able to make cronicle work on windows with relatively small changes. I started a discussion if you need more details
https://github.com/jhuckaby/Cronicle/discussions/579
